baybe/constraints/conditions.py

```diff
@@ -83,6 +105,17 @@ def evaluate(self, data: pd.Series) -> pd.Series:
             A boolean series indicating which elements satisfy the condition.
         """
 
+    @abstractmethod
+    def to_polars(self, input: pl.Expr) -> pl.Expr:
+        """Apply the condition to a Polars expression.
+
+        Args:
+            input: Input expression, for instance column selection etc.
+
+        Returns:
+            A expression that can be passed to filter rows that satisfy condition.
```

Comment:
Should be "An expression", but I also wonder about the second half of the sentence. "to filter rows": what is that supposed to mean? Removing those rows? I guess this is wrong? Do you mean something like "to filter down to rows ..."? Not sure how this function is supposed to be designed, but intuitively it would make sense to me that it should filter down to rows that are kept.

Reply:
I changed it, the description of where this expression is used and how is not really to be written here imo